packs/pathfinder-bestiary-2/ravener.json

```diff
@@ -675,13 +675,17 @@
                 },
                 "rules": [
                     {
-                        "key": "Aura",
-                        "radius": 90,
-                        "slug": "cowering-fear",
-                        "traits": [
-                            "emotion",
-                            "fear",
-                            "mental"
+                        "itemType": "action",
```

Comment:
Does the frightful presence ability need to be added?

Reply:
The creature already has frightful presence. This just adds Cowering Fear to the description of that feature, instead of drawing another circle for it.

![image](https://github.com/user-attachments/assets/becf2f2e-3f51-44d7-a099-964c7a99460e)